### ISSUE TITLE
Game error on Reality Shift trying to manifest non creature cards like Life // Death

### ISSUE THREAD

**Issue by JayDi85** [magefree/mage#11573]
If you try to manifest mdf card like [[Life // Death]] by [[Reality Shift]]'s ability then game will raise an error. Server logs:

```
java.lang.IllegalArgumentException: Wrong code usage: can't create permanent card from split or mdf: Life // Death
	at mage.game.permanent.PermanentCard.<init>(PermanentCard.java:53)
    	at mage.game.ZonesHandler.maybeRemoveFromSourceZone(ZonesHandler.java:359)
    	at mage.game.ZonesHandler.lambda$moveCards$0(ZonesHandler.java:107)
    	at java.util.ArrayList.removeIf(ArrayList.java:1413)
    	at mage.game.ZonesHandler.moveCards(ZonesHandler.java:107)
    	at mage.players.PlayerImpl.moveCards(PlayerImpl.java:4645)
    	at mage.abilities.effects.keyword.ManifestTargetPlayerEffect.apply(ManifestTargetPlayerEffect.java:66)
    	at mage.cards.r.RealityShiftEffect.apply(RealityShift.java:67)
    	at mage.abilities.AbilityImpl.resolveMode(AbilityImpl.java:192)
    	at mage.abilities.AbilityImpl.resolve(AbilityImpl.java:179)
    	at mage.game.stack.Spell.resolve(Spell.java:291)
    	at mage.game.GameImpl.resolve(GameImpl.java:1734)
    	at mage.game.GameImpl.playPriority(GameImpl.java:1659)
    	at mage.game.turn.Step.priority(Step.java:67)
    	at mage.game.turn.Phase.playStep(Phase.java:204)
    	at mage.game.turn.Phase.play(Phase.java:90)
    	at mage.game.turn.Turn.play(Turn.java:137)
    	at mage.game.GameImpl.playTurn(GameImpl.java:1112)
    	at mage.game.GameImpl.play(GameImpl.java:1019)
    	at mage.game.GameImpl.start(GameImpl.java:995)
    	at mage.server.game.GameWorker.call(GameWorker.java:34)
    	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
    	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
    	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
```

**Comment by xenohedron** [comment]
See #10608 and associated PR #10803 that should fix it

**Comment by JayDi85** [comment]
Fixed by 11ddfa0087519b434eb594e20285c8fdba0744d9, part of #11873
